.circleci/config.yml

```diff
@@ -61,55 +62,18 @@ jobs:
     steps:
       - add_ssh_keys
       - checkout
-      - attach_workspace:
-          at: /tmp/workspace
       - run:
           name: Configure
           command: hokusai configure --kubectl-version 1.6.3 --s3-bucket artsy-citadel --s3-key k8s/config --platform linux
+      - run:
+          name: Build Assets
+          command: hokusai dev run 'yarn publish_assets'
       - run:
           name: Asset Manifest
-          command: hokusai production env set ASSET_MANIFEST=$(cat /tmp/workspace/manifest.json)
+          command: hokusai production env set ASSET_MANIFEST=$(cat manifest.json)
       - run:
           name: Deploy
           command: hokusai production deploy $CIRCLE_SHA1
-  deploy_heroku_staging:
-    docker:
-      - image: circleci/node:8-stretch-browsers
-    steps:
-      - add_ssh_keys:
-          fingerprints:
-            - "ed:85:78:e9:d9:4b:3e:e7:78:15:1a:a4:6a:e7:b8:32"
-      - checkout
-      - setup_remote_docker
-      - run: DEPLOY_ENV=staging .circleci/heroku_setup.sh
-      - run:
-          name: "Deploy to staging"
-          command: "yarn install && DEPLOY_ENV=staging .circleci/heroku_deploy.sh && git push git@github.com:artsy/force.git $CIRCLE_SHA1:staging --force"
```

Comment:
Since we're losing this `git push ...staging`, we'll need to replace it elsewhere.

It might be even better to just use latest `hokusai` and add the `--git-remote` parameter to update `staging` and `production` tags on each deploy/promote, but then the README should be updated to refer to the new tags.

Reply:
Do we have a deploy that does this? Looking for an example.

Reply:
https://github.com/artsy/gravity/blob/a08ef15b0a0569f4b9683d99ba8d5c3083ffb6cc/.circleci/config.yml

Reply:
Added it I think.